casual/casual-jca/src/main/java/se/laz/casual/jca/work/StartInboundServerWork.java

```diff
@@ -92,6 +100,25 @@ private void waitForInboundStartupServices()
         log.info(() -> "All startup services registered.");
     }
 
+    private void logInitialStartupServices(List<String> startupServices)
+    {
+        if(ConfigurationService.getInstance().getConfiguration().getInbound().getStartup().getMode() == Mode.DISCOVER)
+        {
+            log.info(() -> "Waiting for " + startupServices.size() + " services to be registered before inbound starts.");
```

Comment:
This is a change from the current logging behaviour, previously we always logged this line.

Reply:
Yes it does.
However, does that logging make any sense if startup mode is not discover?
That is, is it a valid case to have a list of startup services with any other mode than discover?
If so, what are they?

Reply:
Trigger mode - in that case there is 1 service it is waiting for, this would remove the logging showing it is waiting for that.
 
 https://github.com/casualcore/casual-java/blob/dev/casual/casual-api/src/main/java/se/laz/casual/config/Startup.java#L33-L44
 
Perhaps if we add a line saying "Inbound startup mode X".
Then remove the checks for which mode for the logging as they make sense.

```
Inbound startup mode IMMEDIATE
Waiting for 0 services...
All startup services registered.
```
```
Inbound startup mode TRIGGER
Waiting for 1 services...
Name...
All startup services registered.
```
```
Inbound startup mode DISCOVER
Waiting for x services...
Name...
Waiting for x-2 services...
Name...
All startup services registered.
```